src/main/resources/anniv5Resources/localization/eng/warriorpack/Cardstrings.json

```diff
@@ -0,0 +1,44 @@
+{
+  "${ModID}:Sharpening": {
+    "NAME": "Sharpening",
+    "DESCRIPTION": "Whenever you play an Attack, increase its damage by !M! this combat."
+  },
+  "${ModID}:WorkHammer": {
+    "NAME": "Work Hammer",
+    "DESCRIPTION": "Deal !D! ${ModID}:Front damage. NL Both you and the enemy gain !B! Block."
```

Comment:
Does not actually describe its effect correctly. Order of operations is important here.
Also very much an iron wave, please rethink this card. Maybe something scaling? Deal FrontDamage, if the enemy has block increase this cards damage by amount this combat?

Reply:
Iron wave says deal damage then gain block, the player still gains block before. Also your idea overlaps the power I think ? Could do other stuff however it kind of allows chain Front Damage, is it not an acceptable case of iron waving ?
Will try to find smth if it really can't stay the same

Reply:
Actual effect is more like "Gain Block. Deal front Damage. Give enemy Block." Players needs to know they gain Block before damage to know if they can attack a thorns enemy.

In the end, this is another Ironwave but it does synergize with your pack mechanics enough that you can leave it if you really want it to stay.

Reply:
https://static.wikia.nocookie.net/slay-the-spire/images/9/9e/IronWave.png/revision/latest?cb=20181016210817
Nah, it says block, then damage,